float64 tank_1
float64 tank_2
uint32 status  # 0: OK, 1: Warning, 2: Error

